.github/workflows/unit-and-integration-tests-unified.yml

```diff
@@ -16,10 +16,22 @@
 name: Unit & Integration tests CI
 on:
   push:
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
+      - 'helm/**'
+      - 'integration-tests*/**'
+      - 'web-console/**'
     branches:
       - master
       - /^\d+\.\d+\.\d+(-\S*)?$/ # release branches
   pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
+      - 'helm/**'
+      - 'integration-tests*/**'
```

Comment:
We would want to run ITs in case of changes in these paths, Isn't it?

Reply:
Fixed.

Reply:
I guess we can also add extra `paths-ignore` on individual workflows, suppose `'integration-tests*/**'` in `unit-tests.yml`, but that might also not let ITs run as per the current flow. We could alter the flow to run ITs if unit tests are either successful or skipped. But it's complex and can be done as part of a separate PR.